components/core/src/Query.hpp

```diff
@@ -224,7 +214,7 @@ class Query {
     bool m_search_string_matches_all;
```

Comment:
Not initialized, used to be set to false in old initializer list. Probably want to set to false here?